src/openforms/registrations/contrib/objects_api/typing.py

```diff
@@ -15,6 +15,7 @@ class _BaseRegistrationOptions(TypedDict, total=False):
     objects_api_group: Required[ObjectsAPIGroupConfig]
     objecttype: Required[UUID]
     objecttype_version: Required[int]
+    update_existing_object: bool
```

Comment:
Depending on whether we add a data migration, this might be `Required[bool]`

Reply:
I don't see the reason for a data migration for this, maybe Sergei has a different opinion though..

Reply:
fine as is, missing/`None` is equivalent to explicit `False` which should be the default

Reply:
actually, looking at https://github.com/open-formulieren/open-forms/pull/4538#discussion_r1687768751 - if the serializer field indeed has a `default=` specified, then you can make the type hint `Required[...]` since the serializer will ensure the key is always provided, even if it was not initially provided.

Reply:
Ah yes it is `Required` even without the migration, as this typed dict represents the data from the serializer (so with the default value)